### PR TITLE
Nodejs part2

### PR DESCRIPTION
**Pull request by ollyyasinski**
image

To login with a new user: 
image

To delete/edit news
 1) copy token from login
 2) use as authentication header for the request
image

### CODE REVIEW

File: nodeJS/authentication/facebook-strategy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const passport = require("passport");
+
+FacebookStrategy = require("passport-facebook").Strategy;
+
+passport.use(
+  new FacebookStrategy(
+    {
+      clientID: "584264322374454",
```
Comment (GlebShaucou) [inline nodeJS/authentication/facebook-strategy.js]: Why you don't use .env for that?

File: nodeJS/models/news-model.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  name: { type: mongoose.SchemaTypes.String, required: true }
+};
+
+const collectionName = "news";
+const newsSchema = mongoose.Schema(schema);
+const News = mongoose.model(collectionName, newsSchema);
+
+News.deleteMany({}, () => console.log("collection cleared"));
+
+News.create(newsJSON, err => {
```
Comment (GlebShaucou) [inline nodeJS/models/news-model.js]: It is a good practice to notify that some data were created successfully.

Comment (ollyyasinski) [reply nodeJS/models/news-model.js]: logging added

File: nodeJS/authentication/facebook-strategy.js (hunk 1 above)
Comment (ollyyasinski) [reply nodeJS/authentication/facebook-strategy.js]: Thanks, fixed

**Pull request merged**